Yoti.Auth.Sandbox/DocScan/Request/Resource/Liveness/ZoomFrame.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net.Http.Headers;
+
+namespace Yoti.Auth.Sandbox.DocScan.Request.Resource.Liveness
+{
+    public class ZoomFrame : SandboxMedia
```

Comment:
This should live with the integration tests (web client)

Reply:
Moved in ddbb848